index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <h1 class="center" id="title">freeCodeCamp Survey Form</h1>
+  <p class="center" id="description">Thank you for taking the time to help us improve the platform
+  </p>
+  <div class="center"> 
```

Comment:
I think it would be better to use semantic tags. What do you think if `<div class="center">` is replaced with `<main class="center">`